spec.html

```diff
@@ -14306,14 +14344,18 @@ <h2>Supplemental Syntax</h2>
 
         ObjectAssignmentPattern[Yield, Await] :
           `{` `}`
+          `{` AssignmentRestProperty[?Yield, ?Await]? `}`
           `{` AssignmentPropertyList[?Yield, ?Await] `}`
-          `{` AssignmentPropertyList[?Yield, ?Await] `,` `}`
+          `{` AssignmentPropertyList[?Yield, ?Await] `,` AssignmentRestProperty[?Yield, ?Await]? `}`
 
         ArrayAssignmentPattern[Yield, Await] :
           `[` Elision? AssignmentRestElement[?Yield, ?Await]? `]`
           `[` AssignmentElementList[?Yield, ?Await] `]`
           `[` AssignmentElementList[?Yield, ?Await] `,` Elision? AssignmentRestElement[?Yield, ?Await]? `]`
 
+        AssignmentRestProperty[Yield, Await] :
+          `...` DestructuringAssignmentTarget[?Yield, ?Await]
```

Comment:
In the spec text on the repo this [`AssignmentPropertyList` production](https://github.com/tc39/proposal-object-rest-spread/blob/master/Spec.html#L75) has `[Yield, Await]` but I'm sure it's meant to be `[?Yield, ?Await]`. I've made it `[?Yield, ?Await]` here to expedite things, but otherwise I'm happy to raise an issue in the repo and resolve it there before moving forward with this PR.